### ISSUE TITLE
Fix a11y issues

### ISSUE THREAD

**Issue by bmarty** [vector-im/riotX-android#585]
Some has been fixed here #581, but there are still issues.

Global:
- [ ] Missing contentDescription (lint): either add the contentDescription, or add `android:importantForAccessibility="no"`. Also this lint error should be treated as an error to avoid regression on this (update file `lint.xml`)
- [ ] Missing labelFor (lint): same work to do
- [ ] KeyboardInaccessibleWidget (lint): same work to do

Other observed issues:
- [ ] On the home tab (catchup), #581 has improved the navigation with TalkBack, but the FAB is still focused after the element of the list. It's ok on the tabs Direct messages and Rooms

- [ ] Consider setting up a11y test, as per https://developer.android.com/guide/topics/ui/accessibility/testing#automated

Ref: 
- https://developer.android.com/guide/topics/ui/accessibility/apps
- https://developer.android.com/guide/topics/ui/accessibility/testing
